modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/internal/jaxrs/application/FDSApplication.java

```diff
@@ -179,6 +183,39 @@ public Response saveActiveFDSViewSettings(
 		).build();
 	}
 
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/fds/{fdsName}/custom-views")
+	@POST
+	public Response saveFDSCustomView(
```

Comment:
I guess in some moment it would be good to move this to a REST API module using the REST Builder

Reply:
I completely agree! It makes no sense for this to be in a `-taglib` module. I added this to the [technical improvements checklist](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2002681985/FDS+technical+improvements+checklist#FDS-API).

Reply:
Perfect. ¡Thank you!